### PR TITLE
Add wording for enterprise page

### PR DESCRIPTION
**Pull request by fevisera**
### Description of the change

Add wording for enterprise page.

### Applicable issues

- fixes #

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [ ] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [ ] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Review by agomezmoron — APPROVED** [review]

File: bitnami/discourse/README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Discourse is an open source discussion platform with built-in moderation and gov
 
 Trademarks: This software listing is packaged by Bitnami. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement.
 
+Looking to use Discourse## TL;DRreg; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```
Comment (carrodher) [inline bitnami/discourse/README.md:11]:
```suggestion
Looking to use Discourse&reg; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```

Comment (carrodher) [inline bitnami/discourse/README.md:11]:
```suggestion
Looking to use Discourse&reg; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```

File: bitnami/influxdb/README.md [hunk 2]
```diff
@@ -8,6 +8,8 @@ InfluxDB(TM) is an open source time-series database. It is a core component of t
 
 InfluxDB(TM) is a trademark owned by InfluxData, which is not affiliated with, and does not endorse, this site.
 
+Looking to use InfluxDB## TL;DRtrade; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```
Comment (carrodher) [inline bitnami/influxdb/README.md:11]:
```suggestion
Looking to use InfluxDB&trade; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```

File: bitnami/minio/README.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ MinIO(R) is an object storage server, compatible with Amazon S3 cloud storage se
 
 Disclaimer: All software products, projects and company names are trademark(TM) or registered(R) trademarks of their respective holders, and use of them does not imply any affiliation or endorsement. This software is licensed to you subject to one or more open source licenses and VMware provides the software on an AS-IS basis. MinIO(R) is a registered trademark of the MinIO Inc. in the US and other countries. Bitnami is not affiliated, associated, authorized, endorsed by, or in any way officially connected with MinIO Inc. MinIO(R) is licensed under GNU AGPL v3.0.
 
+Looking to use Bitnami Object Storage based on MinIO## TL;DRreg; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```
Comment (carrodher) [inline bitnami/minio/README.md:11]:
```suggestion
Looking to use Bitnami Object Storage based on MinIO&reg; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```

File: bitnami/mongodb-sharded/README.md [hunk 4]
```diff
@@ -8,6 +8,8 @@ MongoDB(R) is an open source NoSQL database that uses JSON for data storage. Mon
 
 Disclaimer: The respective trademarks mentioned in the offering are owned by the respective companies. We do not provide a commercial license for any of these products. This listing has an open-source license. MongoDB(R) is run and maintained by MongoDB, which is a completely separate project from Bitnami.
 
+Looking to use MongoDB## TL;DRreg; Sharded in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```
Comment (carrodher) [inline bitnami/mongodb-sharded/README.md:11]:
```suggestion
Looking to use MongoDB&reg; Sharded in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```

File: bitnami/moodle/README.md [hunk 5]
```diff
@@ -8,6 +8,8 @@ Moodle(TM) LMS is an open source online Learning Management System widely used a
 
 Disclaimer: The respective trademarks mentioned in the offering are owned by the respective companies. We do not provide commercial license of any of these products. This listing has an open source license. Moodle(TM) LMS is run and maintained by Moodle HQ, that is a completely and separate project from Bitnami.
 
+Looking to use Bitnami LMS powered by Moodle## TL;DRtrade; LMS in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```
Comment (carrodher) [inline bitnami/moodle/README.md:11]:
```suggestion
Looking to use Bitnami LMS powered by Moodle&trade; LMS in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```

File: bitnami/redis/README.md [hunk 6]
```diff
@@ -8,6 +8,8 @@ Redis(R) is an open source, advanced key-value store. It is often referred to as
 
 Disclaimer: Redis is a registered trademark of Redis Ltd. Any rights therein are reserved to Redis Ltd. Any use by Bitnami is for referential purposes only and does not indicate any sponsorship, endorsement, or affiliation between Redis Ltd.
 
+Looking to use Redis## TL;DRreg; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```
Comment (carrodher) [inline bitnami/redis/README.md:11]:
```suggestion
Looking to use Redis&reg; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```

File: bitnami/sonarqube/README.md [hunk 7]
```diff
@@ -8,6 +8,8 @@ SonarQube(TM) is an open source quality management platform that analyzes and me
 
 Trademarks: This software listing is packaged by Bitnami. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement. SonarQube is a registered trademark of SonarSource SA.
 
+Looking to use SonarQube## TL;DRtrade; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```
Comment (carrodher) [inline bitnami/sonarqube/README.md:11]:
```suggestion
Looking to use SonarQube&trade; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```

File: bitnami/discourse/README.md (hunk 1 above)
Comment (fevisera) [reply bitnami/discourse/README.md]: This suggestion is duplicated. I will mark this one as resolved.

BTW, thanks for the reviews!

**Comment by fevisera** [comment]
@carrodher / @agomezmoron I lost your approval after applying the review suggestions.
Could you please check it again? Thanks!

**Review by carrodher — APPROVED** [review]

**Pull request merged**